client/commands/v2/pysa_server.py

```diff
@@ -11,7 +11,12 @@
 
 import asyncio
 import logging
+from collections import defaultdict
+from pathlib import Path
+from typing import List, Sequence, Dict, Set
 
+from ....api import query, connection as api_connection
+from ....api.connection import PyreQueryError
```

Comment:
I would avoid relying on things in the `api` package -- those are considered "external" APIs that are currently going through multiple redundant levels of indirections (e.g. shelling out to `pyre` executable, which may or may not match the trunk version). For v2 command, there's no need to shell out to anything: we could query the background Pyre server by invoking command-internal logic directly. 

Querying the Pyre server in pure Python can be done with `commands.v2.query.query_server`:
```
from . import query

try:
  socket_path = Path(self.pyre_arguments.base_arguments.log_path)  # You probably need a rebase to get `base_arguments` to appear
  response_json = query.query_server(socket_path, "validate_taint_models()").payload
except query.InvalidQueryResponse:
  ...
```
Some followup logic may be needed to extract useful info from the `response_json` dict.